NEWS.md

```diff
@@ -1,3 +1,22 @@
+flux-accounting version 0.9.0 - 2021-06-30
+------------------------------------------
+
+#### Fixes
+
+* Fixed bug where users couldn't be added due to a broken function header (#140)
```

Comment:
@cmoussa1: Very our work and news items!

Super minor knit. We have been using present tense, for example. 

```suggestion
* Fix bug where users couldn't be added due to a broken function header (#140)
* ```

Reply:
Other items below as well.

Reply:
Oh! I'm sorry about that. I just pushed up a change to change these to present-tense verbs. I'm looking at the release notes from `v0.8.0` are also in the past tense, so maybe I can post a small fix PR later to change the tense of those notes as well.

Reply:
Thanks @cmoussa1. 

> maybe I can post a small fix PR later to change the tense of those notes as well.
That shouldn't be necessary I think. This is just minor enough.